### PR TITLE
added steps in k8s readme on how to enable http for cluster

### PR DESCRIPTION
**Pull request by DivyaMandati**
added steps in k8s readme on how to enable http for cluster

### CODE REVIEW

File: k8s/007-projects/001-custom/all-ingress.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ metadata:
 spec:
   rules:
   # - host: aks-helloworld.<CLUSTER_SPECIFIC_DNS_ZONE>
-  - host: apps.8e1005bc066142c3b353.southafricanorth.aksapp.io
+  - host: apps.d8db806e5a8b43dca905.southafricanorth.aksapp.io
```
Comment (techslateramu) [inline k8s/007-projects/001-custom/all-ingress.yaml:12]: Why the host name to change? Is this in sync with our demo ??

**Review by techslateramu — COMMENTED** [review]
Please address the comment

File: k8s/007-projects/001-custom/all-ingress.yaml (hunk 1 above)
Comment (DivyaMandati) [reply k8s/007-projects/001-custom/all-ingress.yaml:12]: I think i have changed it to mine when i ran ingress should i change it to previous?

Comment (techslateramu) [reply k8s/007-projects/001-custom/all-ingress.yaml:12]: Yes pls change it back

Comment (DivyaMandati) [reply k8s/007-projects/001-custom/all-ingress.yaml]: Done changes as requested

**Review by techslateramu — APPROVED** [review]

**Pull request merged**